### PR TITLE
add authorization for zookeeper group

### PR DESCRIPTION
**Pull request by sunbufu**
Hi guy, I add authorization for zk group. Can you review it, Thanks.

### CODE REVIEW

**Comment by nzomkxia** [comment]
Thank you for your pull request, but the `dubbo.configcenter` is the new config for dubbo 2.7, please keep it

**Comment by sunbufu** [comment]
How about this.
```
dubbo.registry.group=dubbo
dubbo.configcenter.username=username
dubbo.configcenter.password=password
dubbo.configcenter=zookeeper://127.0.0.1:2181
dubbo.apollo.token=e16e5cd903fd0c97a116c873b448544b9d086de9
dubbo.apollo.appId=test
dubbo.apollo.env=dev
dubbo.apollo.cluster=default
dubbo.apollo.namespace=dubbo
```

**Comment by nzomkxia** [comment]
Since the first version will be released soon, the `develop` branch will not accept new feature, I'll deal it later, thanks

**Comment by sunbufu** [comment]
So, what should i do next?

**Comment by sunbufu** [comment]
Hi, maybe I didn't describe it clearly before. Our zookeeper be divide different group for different develop department, and different group have different password. For example, I have a zookeeper group named "dubbo_group_a" for  "a" department, and this group have this password "password_a". We know dubbo use  "dubbo" group for register by default. and sometime we do not use "dubbo" group. So i think dubbo admin should  support it.

**Comment by nzomkxia** [comment]
> Hi, maybe I didn't describe it clearly before. Our zookeeper be divide different group for different develop department, and different group have different password. For example, I have a zookeeper group named "dubbo_group_a" for "a" department, and this group have this password "password_a". We know dubbo use "dubbo" group for register by default. and sometime we do not use "dubbo" group. So i think dubbo admin should support it.

* you can change zookeeper group here: https://github.com/apache/incubator-dubbo-ops/blob/develop/dubbo-admin-backend/src/main/resources/application.properties#L18  
* authorization for zk is a good feature, I'll merge it after the first release

File: dubbo-admin-backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -16,6 +16,8 @@
 #
 
 dubbo.registry.group=dubbo
+dubbo.configcenter.username=username
+dubbo.configcenter.password=password
```
Comment (nzomkxia) [inline dubbo-admin-backend/src/main/resources/application.properties]: please change the style of configcenter to config-center, thanks

Comment (sunbufu) [reply dubbo-admin-backend/src/main/resources/application.properties]: Hi, I update this properties. But when I push it and run CI, I get an error '`The Travis CI build failed`'. and details is '`Failed to retrieve POM for org.apache.maven.surefire:surefire-junit4:jar:2.21.0: Could not transfer artifact org.apache.maven.surefire:surefire-junit4:pom:2.21.0 from/to central (https://repo.maven.apache.org/maven2): Connection reset`'. I do not change any POM file. Can you help me for this ? Thank you.

**Comment by sunbufu** [comment]
Please review this PR again, thank you.

**Comment by nzomkxia** [comment]
thank you  I'll merge it

**Comment by sunbufu** [comment]
Thank you for merge this PR. Please email me when you need help;

**Comment by sunbufu** [comment]
Thank you for merge this PR. Please email me when you need help.

**Pull request merged**